x-pack/plugins/search_indices/public/components/indices/details_page.tsx

```diff
@@ -70,6 +78,25 @@ export const SearchIndexDetailsPage = () => {
     setDocumentsLoading(isInitialLoading);
     setDocumentsExists(!(!isInitialLoading && indexDocuments?.results?.data.length === 0));
   }, [indexDocuments, isInitialLoading, setDocumentsExists, setDocumentsLoading]);
+
+  useEffect(() => {
+    chrome.docTitle.change(indexName);
+
+    if (serverless) {
```

Comment:
We decided for now to do this route until the search_navigation plugin comes in where it will handle for the differences between stack and serverless.